grove/accel3xdigital/accel3xdigital.go

```diff
@@ -0,0 +1,219 @@
+// Package accel3xdigital allows developers to read x,y,z and acceleratation.
+// Currently this library doesn't support any of the interrupts:  Front/Back Interrupt,
+// Up/Down/Left/Right Interrupt, Tap Detection Interrupt, GINT (real-time motion tracking), Shake on X-axis, Shake on Y-axis, and
+// Shake on Z-axis.
+package accel3xdigital
+
+import (
+	"errors"
+	"fmt"
+
+	"golang.org/x/exp/io/i2c"
+	"golang.org/x/exp/io/i2c/driver"
+)
+
+var (
+	// ErrNotReady warns the user that the device isn't not (yet) ready
+	ErrNotReady = errors.New("device is not ready")
+)
+
+// Accel3xDigital reresents the Grove 3-Axis Digital Accelerometer(±1.5g)
+type Accel3xDigital struct {
+	Device *i2c.Device
+	State  *State
+	// TapEnabled lets devs know if the feature is on or off (default)
+	TapEnabled bool
+}
+
+// New connects to the passed driver and sets things up.
+// At this point the sensor will sample 32 times a second and will store its information in registries you can read from
+// by calling update.
+// Note that by default the tap detection is not on. You need to enable this feature manually.
+func New(o driver.Opener) (*Accel3xDigital, error) {
```

Comment:
Should we s/New/Open all around? New is used to construct instances, Open is the terminology in Go if it opens a resource that needs to closed. I renamed New to Open at https://godoc.org/github.com/goiot/devices/dotstar#Open.